File: python/5.py
from math import log, floor

def smallestCommonMultiple(max: int) -> int:
    primes = [2, 3, 5, 7, 11, 13, 17, 19]
    product = 1

    for i in primes:
        product *= i ** floor(log(max) / log(i))

    return product

print(smallestCommonMultiple(20))

File: typescript/5.ts
const primes = [2, 3, 5, 7, 11, 13, 17, 19];
let product = 1;

for (const i of primes) {
    product *= i ** Math.floor(Math.log(20) / Math.log(i));
}

console.log(product);
// 232792560

export {}
